Adding documentation for NHibernate (#5207)
Particular/docs.particular.net: nservicebus/tools/migrate-to-native-delivery.md

`SELECT * FROM MigrationsEntity`

> To list all of the timeouts that were supposed to be migrated run the following query

Reviewer: ```suggestion
To list all the timeouts that were staged for migration, run the following query
```
